Client/Application/Delegates/AppDelegate.swift

```diff
@@ -392,6 +392,7 @@ class AppDelegate: UIResponder, UIApplicationDelegate, UIViewControllerRestorati
 
         if let profile = self.profile {
             profile.reopen()
+            setUpWebServer(profile)
```

Comment:
Will be reverted when: https://github.com/brave/brave-ios/pull/2945 - is merged

Reply:
merged this, feel free to revert

Reply:
done!